### DISCUSSION TITLE
how can v avoid dlang fail

### DISCUSSION THREAD

**Discussion by legendsf** [vlang/v#17589] [Ideas]
how can vlang support by big company not just community

without company vlang cannot have big influence

study rust and go community work stlye

**Comment by i582** [comment]
What do you mean by D fail?

**Reply by legendsf** [reply]
https://devm.io/software-architecture/which-language-will-replace-c-ds-lead-architect-weighs-in-122161

http://qiusuoge.com/14748.html


i mean fail because dlang birth 1999,  earier than golang rust , but until now so few people use it 

dlang have a good performance than go, and have autofree gc and mannul free gc, like vlang 

i think vlang will much sucess with benifite belows:

1. company support not just community
2. [Built-in] high-quality offical stdlib like golang does， and have pandas numpy liked stdlib, for the AI
3. cloud native and AI support

**Reply by i582** [reply]
Thanks for sharing!

**Comment by JalonSolov** [comment]
1. V has corporate sponsors.  If you want more "company" than that, perhaps the company you own would like to sponsor it as well?

2. V has a good stdlib, as well as [vsl](https://github.com/vlang/vsl) and [vtl](https://github.com/vlang/vtl)

3. Cloud native... not yet.  AI, yes, see the libs listed in # 2.

Perhaps you should do more research before telling V what it needs.

You could also contribute code if you see something that's missing.

**Reply by legendsf** [reply]
thanks a lot ！

**Comment by rraammiinn** [comment]
I don't know about others but for me it's enough that d is unnecessarily ugly .
